packages/libfcl/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/flexible-collision-library/fcl"
+TERMUX_PKG_DESCRIPTION="flexible collision library"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="Pooya Moradi <pvonmoradi@gmail.com>"
+TERMUX_PKG_VERSION="0.7.0"
+TERMUX_PKG_SRCURL="https://github.com/flexible-collision-library/fcl/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=90409e940b24045987506a6b239424a4222e2daf648c86dd146cbcb692ebdcbc
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_FORCE_CMAKE=true
+TERMUX_PKG_BUILD_DEPENDS="eigen, libccd, octomap"
```

Comment:
libccd and octomap are runtime dependencies (libfcl.so are linked to them) so they need to be in `TERMUX_PKG_DEPENDS`. Are you sure eigen is needed? Build doesn't mention eigen ever so I am not sure it is used for anything

Reply:
Yes, It is [declared in its CMakeLists](https://github.com/flexible-collision-library/fcl/blob/master/CMakeLists.txt#L171)